app/model/album.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var app = express();
+var fs =require('fs');
+
+Album = function() {
+  this.albums = [];
+  this.loadAll();
+}
+
+Album.prototype.get =function(id) {
+  if (!id) {return false;}
+  return this.albums[id];
+}
+
+Album.prototype.countAlbums = function(id)
+{
```

Comment:
This is inconsistent styling so you should either fix this one or change the other opening curly braces.